doc/modules/cross_validation.rst

```diff
@@ -240,16 +240,20 @@ prediction that was obtained for that element when it was in the test set. Only
 cross-validation strategies that assign all elements to a test set exactly once
 can be used (otherwise, an exception is raised).
 
-These prediction can then be used to evaluate the classifier::
 
-  >>> from sklearn.model_selection import cross_val_predict
-  >>> predicted = cross_val_predict(clf, iris.data, iris.target, cv=10)
-  >>> metrics.accuracy_score(iris.target, predicted) # doctest: +ELLIPSIS
-  0.973...
+Note on inappropriate usage of cross_val_predict
```

Comment:
Rather than a new section, you should use the warning sphinx directive. Look at other places in the doc where it is used. Either `git grep '\.\. warning'` or search for `.. warning ::` in https://raw.githubusercontent.com/scikit-learn/scikit-learn/7dffa20363d095320ede354edf66f8b6123ba121/doc/developers/contributing.rst.

Reply:
Done.